code/modules/cargo/expressconsole.dm

```diff
@@ -11,18 +14,28 @@
 	interface_type = "CargoExpress"
 
 	var/message
-	var/printed_beacons = 0 //number of beacons printed. Used to determine beacon names.
 	var/list/meme_pack_data
-	var/obj/item/supplypod_beacon/beacon //the linked supplypod beacon
-	var/area/landingzone = /area/station/cargo/storage //where we droppin boys
-	var/podType = /obj/structure/closet/supplypod
-	var/cooldown = 0 //cooldown to prevent printing supplypod beacon spam
-	var/locked = TRUE //is the console locked? unlock with ID
-	var/usingBeacon = FALSE //is the console in beacon mode? exists to let beacon know when a pod may come in
+	/// The linked supplypod beacon
+	var/obj/item/supplypod_beacon/beacon
+	/// Where we droppin boys
+	var/area/landingzone = /area/station/cargo/storage
+	var/pod_type = /obj/structure/closet/supplypod
+	/// If this console is locked and needs to be unlocked with an ID
+	var/locked = TRUE
+	/// Is the console in beacon mode? Exists to let beacon know when a pod may come in
+	var/using_beacon = FALSE
+	/// Number of beacons printed. Used to determine beacon names.
+	var/static/printed_beacons = 0
+	/// Cooldown to prevent beacon spam
+	COOLDOWN_DECLARE(beacon_print_cooldown)
 
 /obj/machinery/computer/cargo/express/Initialize(mapload)
 	. = ..()
 	packin_up()
+	landingzone = GLOB.areas_by_type[landingzone]
+	if (isnull(landingzone))
+		WARNING("[src] couldnt find a Quartermaster/Storage (aka cargobay) area on the station, and as such it has set the supplypod landingzone to the area it resides in.")
+		landingzone = get_area(src)
```

Comment:
Should we be checking for `mapload` here? 
If not, should this be a stack trace instead?